RadixWalletTests/Features/AccountPreferencesTests/AccountPreferencesTests.swift

```diff
@@ -0,0 +1,50 @@
+@testable import Radix_Wallet_Dev
+import XCTest
+
+@MainActor
+final class AccountPreferencesTests: TestCase {
+	func test_hideAccount_flow() async {
```

Comment:
split this test into multiple tests. Dont care about code duplication to much. And please use GIVEN WHEN THEN format, with comment in code as well. 

We should almost always only have ONE single assert! and with name of test reflecting that assert. That makes tests easier to read, maintain and fix once broken. Also makes it clear in CI log which part of an otherwise big test like this which actually failed.

Reply:
remember for `TestStore` you can use `exhaustivity = .off`